tests/unit/onyxTest.ts

```diff
@@ -733,8 +739,9 @@ describe('Onyx', () => {
                         },
                     ],
                 },
-            ]);
-        } catch (error) {
+            ] as unknown as OnyxUpdate[]);
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        } catch (error: any) {
             expect(error.message).toEqual('Invalid value provided in Onyx multiSet. Onyx multiSet value must be of type object.');
         }
```

Comment:
```suggestion
        } catch (error) {
            if (error instanceof Error) {
               expect(error.message).toEqual('Invalid value provided in Onyx multiSet. Onyx multiSet value must be of type object.');
            }
        }
```

Reply:
I changed it, but I think we might need to throw an error if it's not instanceOf Error:
```ts
   if (error instanceof Error) {
                expect(error.message).toEqual('Invalid value provided in Onyx multiSet. Onyx multiSet value must be of type object.');
            } else {
                throw error;
            }
```